feature/gnpsi/otg_tests/sampling_test/README.md

```diff
@@ -0,0 +1,112 @@
+# gNPSI-1: Sampling and Subscription Check
+
+## Summary
+
+The goal is to validate that packet sampling is working as expected, clients can connect to the gNPSI service and receive samples.  
+
+## Procedure
+
+
+### Test 1: Verify samples are exported via gNPSI service. 
+
+1. Enable sFlow and gNPSI on the DUT.
+
+*   Set sample source address, 
+*   Varying sample sizes (256, 512, 1024)
+*   Varying sampling rates (1:32k, 1:1M, 1:4M).
+*   DSCP=32
+
+2. Send Traffic via OTG with different traffic profiles. 
+
+*   Ipv4 and Ipv6
+*   Varying packet sizes (64, 512, 1500)
+
+3. Start the gRPC client and subscribe to the gRPC service on the DUT.
+
+4. Verify the samples received by the client are as per expectations:
+
```

Comment:
Does OTG support gnpsi client and gnpsi packet decode?  If this is an open feature request, please add a TODOs linking to the OTG issues and/or pull requests.  

FWIW, it is possible to get raw packet capture from OTG, but that won't be helpful with encrypted gRPC.